lib/r_sig.rb

```diff
@@ -0,0 +1,21 @@
+# typed: true
+
+module RSig
+  T::Sig::WithoutRuntime.sig { params(ctx: Module, blk: T.proc.bind(T::Private::Methods::DeclBuilder).void).void }
+  def self.sig(ctx, &blk)
```

Comment:
RSig.sig is not bad, but it introduces another top-level module that could potentially conflict with user code or other gems, even if unlikely.

Instead, would it be possible to extend `RSpec::Core::ExampleGroup` with an `rsig` method?